packages/components/src/components/faq/index.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  Children,
+  cloneElement,
+  ComponentProps,
+  ComponentPropsWithoutRef,
+  FC,
+  ReactElement,
+  ReactNode,
+} from 'react';
+import * as RadixAccordion from '@radix-ui/react-accordion';
+import { ChevronDownIcon } from '@radix-ui/react-icons';
+import { cn } from '../../cn';
+import { Anchor } from '../anchor';
+import { Heading } from '../heading';
+import { AttachPageFAQSchema } from './attach-page-faq-schema';
+
+const UnwrapChild: FC<{ children?: ReactNode }> = props => props.children;
+
+const a: FC<ComponentPropsWithoutRef<'a'>> = props => (
+  <Anchor
+    className="hive-focus rounded underline hover:text-blue-700"
+    {...props}
+    href={props.href!}
+  >
+    {props.children}
+  </Anchor>
+);
+
+const h2: FC<ComponentPropsWithoutRef<'h2'>> = props => (
+  <Heading as="h2" size="md" className="basis-1/2" {...props} />
+);
+
+export const FrequentlyAskedQuestions: FC<
+  ComponentProps<typeof AttachPageFAQSchema> & {
+    className?: string;
+    children: ReactElement;
+  }
+> = ({ className, faqPages, children }) => {
+  return (
+    <section
+      className={cn(
+        className,
+        'flex flex-col gap-x-6 gap-y-2 px-4 py-6 text-green-1000 md:flex-row md:px-10 lg:gap-x-24 lg:px-[120px] lg:py-24',
+      )}
+    >
+      <AttachPageFAQSchema faqPages={faqPages} />
+      {cloneElement(children, {
+        components: {
+          a,
+          h2,
```

Comment:
This can't be done with a provider in Server Components world, right?

Reply:
yes, here is code from Yoga how it looks like
<img width="361" alt="image" src="https://github.com/user-attachments/assets/712eb5c3-f5c8-4189-bf8c-da1d5e839742" />